spec/helpers/flexible_section_helper_spec.rb

```diff
@@ -0,0 +1,12 @@
+RSpec.describe FlexibleSectionHelper do
+  include described_class
+
+  describe "#render_flexible_section" do
+    it "contains error items" do
```

Comment:
Minor: I think you could rename this to `it "renders the correct partial for the flexible section"` rather than talking about errors.